bullet/src/KinematicsFeatures.cc

```diff
@@ -52,11 +52,14 @@ FrameData3d KinematicsFeatures::FrameDataRelativeToWorld(
   const btVector3 pos = trans.getOrigin();
   const btMatrix3x3 mat = trans.getBasis();
 
-  auto eigenMat = convert(mat);
-  auto eigenVec = convert(pos);
-
-  data.pose.linear() = eigenMat;
-  data.pose.translation() = eigenVec;
+  const Eigen::Isometry3d poseIsometry =
+    ignition::math::eigen3::convert(linkInfo->inertialPose.Inverse());
```

Comment:
@azeey Is this correct? Simulation does not look like it should when using this changes